modules/tinymce/src/core/main/ts/api/html/DomParser.ts

```diff
@@ -80,154 +78,19 @@ interface DomParser {
   addNodeFilter: (name: string, callback: (nodes: AstNode[], name: string, args: ParserArgs) => void) => void;
   getNodeFilters: () => ParserFilter[];
   filterNode: (node: AstNode) => AstNode;
-  parse: (html: string, args?: ParserArgs) => AstNode;
+  parse: (html: string, args?: ParserArgs) => Node;
```

Comment:
This needs to return an `AstNode` still, so please make sure to either fix that or leave a TODO for when we have the wrapper.

Reply:
Todo added (now with new and improved _correct_ Jira numbers)